domain-ee/ee-ep-merge-app/src/python_src/schema/merge_job.py

```diff
@@ -57,17 +57,16 @@ def __init__(self, *args, **kwargs) -> None:
     class Meta:
         orm_model = model.merge_job.MergeJob
 
-    def error(self, messages):
+    def error(self, message: dict[Any, Any]):
         self.error_state = self.state
         self.state = JobState.COMPLETED_ERROR
-        self.add_message(messages)
+        self.add_message(message)
 
-    def add_message(self, messages):
-        if messages:
+    def add_message(self, message: dict[Any, Any]):
+        if len(message):
```

Comment:
Just a note that a dict bool-evaluates to true iff it contains at least one mapping, so I think you could keep it as `if message`.